Source/SwiftLintFrameworkTests/YamlParserTests.swift

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
Ah, *this* file is superfluous since Yaml is tested in its own project. Please remove.